RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc

```diff
@@ -705,14 +705,13 @@ bool HybridMinimizer::GetMinosError(unsigned int i, double & errLow, double & er
       // print the real number of maxfcn used (defined in MnMinos)
       int maxfcn_used = maxfcn; 
       if (maxfcn_used == 0) { 
-         int nvar = fState.VariableParameters();
-         maxfcn_used = 2*(nvar+1)*(200 + 100*nvar + 5*nvar*nvar);
+	//         int nvar = fState.VariableParameters();
```

Comment:
the whole chunk of code under ```if (PrintLevel() >=1)``` is unused. Please decide to wholly remove it, or comment out everything. Commenting out unused must normally be avoided: however this whole class is full of commented out ```cout```s, and it would require a major cleaning, which could be probably get deferred to a later PR (unless you want to do it now here),

Reply:
"comment out everything" is what I pushed saturday morning and wasn't fine (?)

Reply:
Yes.... It was in view of a more complete cleaning of this code: I agree that we can defer it by now.